src/Application.php

```diff
@@ -201,13 +201,13 @@ public function pipe($path, $middleware = null)
     {
         // Lazy-load middleware from the container when possible
         $container = $this->container;
-        if (null === $middleware && is_string($path) && $container && $container->has($path, true)) {
+        if (null === $middleware && is_string($path) && $container && $container->has($path)) {
```

Comment:
This change has be do on #194. This PR is on top of that one for prevent merge conflicts.